### DISCUSSION TITLE
Make the tldraw embed have focus mode enabled by default

### DISCUSSION THREAD

**Discussion by MaxValue** [outline/outline#5771] [Ideas]
Currently, when embedding a Tldraw document/page and loading the embedding Outline page, the Tldraw embed is shown with all the drawing tools displayed which clutters up the minimalistic/clutterfree Outline experience.
From what I can roughly tell skimming over the Tldraw dev-docs one can embed Tldraw with the focus mode (no drawing tools shown). I admit that I couldn’t find the specific config required to enable focus mode on load.

How do you think about this change?

**Comment by tommoor** [comment]
Seems preferable if possible 👍  – I also don't see any docs on how to trigger this mode from a query string parameter though

**Comment by MaxValue** [comment]
I wrote to their dev community about this on their Discord now, let’s see.
